import sys
try:
    sys.exit()
except SystemExit:
    print("SystemExitの例外処理")